cypress.config.js

```diff
@@ -1,42 +1,43 @@
 // override: false means that we won't override env vars from the command line
 require('dotenv').config({override: false})
+const { defineConfig } = require('cypress')
+const webpackConfig = require('./webpack.config')
+const env = process.env
 
-let env = process.env
-
-module.exports = {
+module.exports = defineConfig({
   chromeWebSecurity: false,
 
   env: {
     ...env,
     COMMAND_PREFIX: "docker compose exec -T php",
-    coverage: false,
+    coverage: false
   },
 
   nodeVersion: "system",
   viewportWidth: 1600,
 
+  defaultCommandTimeout: 10000,
+
   retries: {
     // Configure retry attempts for `cypress run`
-    runMode: 2,
+    runMode: 4,
```

Comment:
Does it really help? I notice that if it consistently fails 2-3 times, then the issue is probably somewhere else 😄 (more retries just make a test run longer)

Reply:
Yepp u r right, made no difference at all. Reverting!

Reply:
Yepp u r right, made no difference at all. Reverting!

Reply:
Well, in the end, I tried a ridicously high retry number (9) and seemed to make a difference.

This one passed at the last retry:
![image](https://github.com/user-attachments/assets/388d6c7b-aee9-4769-a749-0cdd2b1bea3d)

This one in the 5th run:
![image](https://github.com/user-attachments/assets/064ad2d6-de2a-4481-9412-dc6bd268ec34)


:loudspeaker:  So I will let the `runMode: 9` and see how it behaves with some time being enabled.
We can always change back to whatever value at any moment directly at `master`.